extensions/wikia/Email/Controller/WallMessageController.class.php

```diff
@@ -182,7 +182,15 @@ protected function getSubject() {
 	}
 }
 
-class ReplyWallMessageController extends OwnWallMessageController {
+class ReplyWallMessageController extends WallMessageController {
```

Comment:
This was extending `getEmailSpecificFormFields`, but there's no need to. It overrides to the only 2 methods that class has. Extend the base wall message class instead.